bodhi/server/models.py

```diff
@@ -735,6 +735,75 @@ def unpush(self, koji):
 
 
 class Update(Base):
+    """
+    This model represents a package.
+
+    This model uses single-table inheritance to allow for different package types.
+
+    Attributes:
+        title (unicode): The update's title which uniquely identifies the update.
+        autokarma (bool): A boolean that indicates whether or not the update will
+            be automatically pushed when the stable_karma threshold is reached.
+        stable_karma (int): A positive integer that indicates the amount of "good"
+            karma the update must receive before being marked as stable.
+        unstable_karma (int): A positive integer that indicates the amount of "bad"
+            karma the update must receive before being marked as unstable.
+        requirements (unicode): A list of taskotron tests that must pass for this
+            update to be considered stable. Note: This does not appear to actually
+            work and is likely to be replaced by PolicyEngine in the long-term.
+        require_bugs (bool): Indicates whether or not positive feedback needs to be
+            provided for the associated bugs before the update can be considered
+            stable.
+        require_testcases (bool): Indicates whether or not the update requires that
+            positive feedback be given on all associated wiki test cases before the
+            update can pass to stable. If the update has no associated wiki test cases,
+            this option has no effect.
+        notes (unicode): Notes about the update. This is a human-readable field that
+            describes what the update is for (e.g. the bugs it fixes).
+        type (EnumSymbol): The type of the update (e.g. enhancement, bugfix, etc). It
+            must be one of the values defined in :class:`UpdateType`.
+        status (EnumSymbol): The current status of the update. Possible values include
+            'pending' to indicate it is not yet in a repository, 'testing' to indicate it
+            is in the testing repository, etc. It must be one of the values defined in
+            :class:`UpdateStatus`.
+        request (EnumSymbol): The requested status of the update. This must be one of the
+            values defined in :class:`UpdateRequest` or ``None``.
+        severity (EnumSymbol): The update's severity. This must be one of the values defined
+            in :class:`UpdateSeverity`.
+        suggest (EnumSymbol): Suggested action a user should take after applying the update.
+            This must be one of the values defined in :class:`UpdateSuggestion`.
+        locked (bool): Indicates whether or not the update is locked and un-editable.
```

Comment:
Maybe it would be good to mention that this usually happens when the update is going through a state transition by the masher?